starter-code/css/main.css

```
@@ -1,10 +1,91 @@
-/*
-Colors to use:
-dark blue: #06365f
-light blue: #1c5380
-*/
-
-body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+html,body{
+  font-family: helvetica;
+  padding: 0;
   margin: 0;
+  font-size: 16px
+  
 }
+
+.main{
+  position: relative;
+
+}
+.background{
+  background-image: url("../images/bgtile.png");
+  background-size: cover;
+  height: 300px;
+}
+
+.phones{
+  position: absolute;
+  left: 150px;
+  top: 100px;
+
+}
+
+.brand{
+  position: absolute;
+  top: 120px;
+  left: 550px;
+}
+
+.home {
+  height: 40px;
+  width: 90px;
+  background: #8fd2ff;
+  border: 1px solid #1f386d;
+  position: relative;
+  left: 65%;
+  top: 40%;
+}
+
+.log-in{
+
+}
+h2{
+  text-align: justify;
+  color: #1f386d;
+}
+
+.content{
+  background-image: url("../images/frame.png");
+  background-repeat: no-repeat;
+  width: 500px;
+  height: 400px;
+  position: absolute;
+  top: 200px;
+  left: 550px;
+  padding-left: 1%;
```

Esta propiedad puede ayudarte a que la imagen de la sombra siempre se ajuste al tamaño de la caja sin:
```suggestion
  padding-left: 1%;
  background-size: contain;
```